tokio-async-await/examples/src/hyper.rs

```diff
@@ -21,5 +22,12 @@ pub fn main() {
         }).unwrap();
 
         println!("Response: {}", response.status());
+
+        let mut body = response.into_body();
+
+        while let Some(chunk) = await!(body.next()) {
+            let chunk = chunk.unwrap();
+            println!("chunk = {}", str::from_utf8(&chunk[..]).unwrap());
```

Comment:
Could you return `Box<Error>>` and use `?`?

Reply:
I don’t follow?

Reply:
I mean: https://github.com/tokio-rs/tokio/issues/657

Reply:
Probably. I'll merge this as is as it is an improvement over the status quo, but I'd be happy to get a PR to fix it afterwards.